engine/objconv/mesher/to_OgreMesh.cpp

```diff
@@ -189,15 +188,15 @@ static string getMaterialHash( const XML &memfile )
                              memfile.alphatest );
     hash += "-";
 
-    sprintf( tmp, "-%d-", memfile.textures.size() );
+    sprintf( tmp, "-%lu-", memfile.textures.size() );
```

Comment:
Please use `snprintf`:
```
snprintf(tmp, 64*sizeof(char), "-%lu-", memfile.texturse.size());
```

Reply:
helps prevent buffer overflows ;)

Reply:
I want to do that in a separate PR, as part of #32.